services/drupal/web/modules/custom/epa_core/src/EpaCoreLibraryItemAccessControlHandler.php

```diff
@@ -27,20 +27,17 @@ protected function checkAccess(EntityInterface $library_item, $operation, Accoun
       $access = $access->andIf(AccessResult::allowedIfHasPermission($account, $this->entityType->getAdminPermission()));
     }
     
-    // Users can update if they have 'edit paragraph library item' permission
-    // and they own the paragraph; or if they have admin permission.
+    // Users can update if they have the 'edit paragraph library item'
+    // permission and they own the paragraph.
     if ($operation === 'update') {
       $access = $access->allowedIf($library_item->getOwnerId() == $account->id());
       $access = $access->andIf(AccessResult::allowedIfHasPermission($account, 'edit own paragraph library items'));
       $access = $access->orIf(AccessResult::allowedIfHasPermission($account, 'edit paragraph library item'));
-      $access = $access->orIf(AccessResult::allowedIfHasPermission($account, $this->entityType->getAdminPermission()));
```

Comment:
Admin permission should still allow editing all paragraphs, so please don't remove this.